InvenTree/templates/js/translated/helpers.js

```diff
@@ -326,3 +327,24 @@ function setupNotesField(element, url, options={}) {
         });
     }
 }
+
+
+/*
+ * Sanitize a string provided by the user from an input field,
+ * e.g. data form or search box
+ *
+ * - Remove leading / trailing whitespace
+ * - Remove hidden control characters
+ */
+function sanitizeInputString(s, options={}) {
+
+    s = s.trim();
+
+    // Remove ASCII control characters
+    s = s.replace(/[^\x20-\x7E]+/g, '');
```

Comment:
Suggestion: @SchrodingersGat  should we maybe just reduce to utf8?

Reply:
Worked out a much simpler regex for removing non-printable characters